lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -255,6 +261,32 @@
                 },
 
                 /**
+                 *
+                 * The backbone router entry for editing team members, using topic and team IDs.
+                 */
+                editTeamMembers: function (topicID, teamID) {
+                    var self = this;
+                    this.getTopic(topicID).done(function (topic) {
```

Comment:
Sure, I can update here and in editTeam (which is where I took this pattern from).